server/models/Transaction.js

```diff
@@ -305,6 +305,26 @@ export default (Sequelize, DataTypes) => {
     return Transaction.findByPk(this.RefundTransactionId);
   };
 
+  Transaction.prototype.getFeeOnTopTransaction = function () {
+    return this.getPlatformTipTransaction();
+  };
```

Comment:
I don't get the benefit of providing an alias, shouldn't we just call it "platform tip" and always use `getPlatformTipTransaction`?

Reply:
To have a smooth transition, I wanted to express they're the same thing. But you might be right.